tools/developer.html

```diff
@@ -97,10 +98,14 @@ <h3>Markup</h3>
         var start_time = +new Date();
         var result = hljs.getLanguage(language) ? hljs.highlight(language, source, true) : hljs.highlightAuto(source);
         var rendering_time = +new Date() - start_time;
-        editor.find('.hljs').html(result.value);
-        $(".hljs span").each((_,el) => {
-          $(el).attr("data-klass", el.className.replace("hljs-",""))
-        })
+        vue.code = source;
+        // give Vue a moment to update
+        setTimeout(() => {
+          $(".hljs span").each((_,el) => {
+            $(el).attr("data-klass", el.className.replace("hljs-",""))
+          })
+        }, 10);
```

Comment:
Instead of using a timeout, can't this be done in the `Vue()` as part of the lifecycle? (i.e. `mounted`)

This is some very weird jQuery + Vue butchery lol

Reply:
I'll take another look... this wasn't meant to be a perfect example of usage. :)  It would need to be an "onchange" hook on the code attribute of our app, not on mount I don't think... this needs to fire anytime the code is changed.

Reply:
It's not when it's mounted though it's every single time it's updated... I'm not aware that Vue has a "afterRerender" callback, does it?  `$watch` is too early from my understanding of it.

Reply:
Ok, using $nextTick now. :)